netconf_client/ncclient.py

```diff
@@ -104,6 +104,12 @@ def logger():
         """Returns the internally used logger instance (same for all sessions)"""
         return _logger
 
+    def enable_logging(self, level=logging.DEBUG):
+        _logger.setLevel(level)
+
+    def disable_logging(self, level=logging.NOTSET):
+        _logger.setLevel(level)
+
```

Comment:
From what I'm seeing in the docs, using `logging.NOTSET` doesn't actually disable logging per-se.

How do you feel about this instead?

```suggestion
    def set_logger_level(self, level):
        _logger.setLevel(level)

```

Reply:
that looks good to me 👍